### PR TITLE
Update LB4 as current release

### PR DESCRIPTION
**Pull request by dhmlau**
Update LB4 as current release in this page: https://loopback.io/doc/

### CODE REVIEW

File: doc/index.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ Getting Started with Node for:
 
 This site contains documentation for:
 
-- **[LoopBack 4](en/lb4)** - Developer Preview #3 release. <br/> _Check out the next step in the evolution of LoopBack!_
+- **[LoopBack 4](en/lb4)** - current release. <br/> 
+_LoopBack 4 GA (General Availability) has been released in October 2018, read more in the [announcement post](http://strongloop.com/strongblog/loopback-4-ga)_
 - **[LoopBack 3.x](en/lb3)** - Current production release.
```
Comment (bajtos) [inline doc/index.md]: This should say "Active LTS", not "Current".

File: doc/index.md [hunk 2]
```diff
@@ -19,7 +19,8 @@ Getting Started with Node for:
...
 - **[LoopBack 2.x](en/lb2)** - [Long-term support](/doc/en/contrib/Long-term-support.html) release.
```
Comment (bajtos) [inline doc/index.md]: And 2.x is "Maintenance LTS".

**Review by bajtos — COMMENTED** [review]
PTAL 👇

**Comment by dhmlau** [comment]
Sorry to miss the other releases.  I've updated accordingly.  PTAL @bajtos @jannyHou . Thanks!

**Comment by dhmlau** [comment]
@slnode test please

**Comment by dhmlau** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Review by jannyHou — APPROVED** [review]
👍

**Pull request merged**